lib/SchemaMigrator.js

```diff
@@ -153,7 +153,7 @@ var migrationHandlers = {
 function SchemaMigrator(db, req, table, current, proposed) {
     this.db = db;
     this.req = req;
-    this.table = table + '_data';
```

Comment:
It might still be worth to keep a suffix, as we'll need multiple tables per table name for secondary indexes etc. The suffix avoids collisions where a table happens to be named like one of those indexes.